.github/workflows/allthekeeps-testnet.yml

```diff
@@ -49,14 +49,14 @@ jobs:
       - name: Deploy latest subgraph to thegraph.com
         working-directory: ./keep-subgraph
         run: | 
-          CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${{ secrets.CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY }} \
-          ETH_RPC_URL=${{ secrets.ETH_RPC_URL }} \ 
-          SUBGRAPH_DEPLOY_KEY=${{ secrets.SUBGRAPH_DEPLOY_KEY }} \ 
-          SUBGRAPH_SLUG=${{ secrets.SUBGRAPH_SLUG }} \ 
-          KEEP_CORE_VERSION=${{ steps.upstream-builds-query.outputs.keep-core-contracts-version }} \
-          KEEP_ECDSA_VERSION=${{ steps.upstream-builds-query.outputs.keep-ecdsa-contracts-version }} \
-          KEEP_TBTC_VERSION=${{ steps.upstream-builds-query.outputs.tbtc-contracts-version }} \
-            ./deploy-subgraph.sh
+          export CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${{ secrets.CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY }}
+          export ETH_RPC_URL=${{ secrets.ETH_RPC_URL }}
+          export SUBGRAPH_DEPLOY_KEY=${{ secrets.SUBGRAPH_DEPLOY_KEY }}
+          export SUBGRAPH_SLUG=${{ secrets.SUBGRAPH_SLUG }}
+          export KEEP_CORE_VERSION=${{ steps.upstream-builds-query.outputs.keep-core-contracts-version }}
+          export KEEP_ECDSA_VERSION=${{ steps.upstream-builds-query.outputs.keep-ecdsa-contracts-version }}
+          export KEEP_TBTC_VERSION=${{ steps.upstream-builds-query.outputs.tbtc-contracts-version }}
+          ./deploy-subgraph.sh
 
```

Comment:
The way we handle passing env variables to script in other places is by using [`env`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsenv) keyword, something like this:

```
        env:
          CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${{ secrets.CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY }}
          ETH_RPC_URL=${{ secrets.ETH_RPC_URL }}
          SUBGRAPH_DEPLOY_KEY=${{ secrets.SUBGRAPH_DEPLOY_KEY }}
          SUBGRAPH_SLUG=${{ secrets.SUBGRAPH_SLUG }}
          KEEP_CORE_VERSION=${{ steps.upstream-builds-query.outputs.keep-core-contracts-version }}
          KEEP_ECDSA_VERSION=${{ steps.upstream-builds-query.outputs.keep-ecdsa-contracts-version }}
          KEEP_TBTC_VERSION=${{ steps.upstream-builds-query.outputs.tbtc-contracts-version }}
        run:  ./deploy-subgraph.sh
```
Could we try it?

Reply:
Sure, it worked but I had to change `=` with `:`

Reply:
Right, good catch, I missed that during code edit